src/main/resources/Untitled2.sql

```diff
@@ -0,0 +1,150 @@
+WHERE p1.DESCRIPTION LIKE '%a%' OR p1.DESCRIPTION LIKE '%A%' OR p1.description LIKE '%�%'  OR p1.description LIKE '%�%';
+--
+SELECT ORDER_NUM, AMOUNT, COMPANY, CREDIT_LIMIT  FROM ORDERS, CUSTOMERS  WHERE CUST = CUST_NUM;
+SELECT ORDER_NUM, AMOUNT, COMPANY, CREDIT_LIMIT  FROM ORDERS o1 INNER JOIN CUSTOMERS o2  ON o1.cust = o2.cust_num;
+SELECT   order_num,  amount, company, credit_limit FROM (SELECT * FROM orders INNER JOIN customers ON orders.cust = customers.cust_num );
+--
+SELECT NAME, CITY, REGION  FROM SALESREPS, OFFICES  WHERE REP_OFFICE = OFFICE;
+SELECT NAME, CITY, REGION  FROM SALESREPS o1 INNER JOIN OFFICES o2  ON o1.rep_office=o2.office; 
+--
+SELECT CITY, NAME, TITLE  FROM OFFICES, SALESREPS  WHERE MGR = EMPL_NUM;
+SELECT CITY, NAME, TITLE  FROM OFFICES o1 INNER JOIN SALESREPS o2 ON o1.mgr = o2.empl_num; 
+--
+SELECT CITY, NAME, TITLE  FROM OFFICES, SALESREPS    WHERE MGR = EMPL_NUM AND  TARGET > 600.00;
+SELECT CITY, NAME, TITLE  FROM OFFICES o1 INNER JOIN SALESREPS o2  ON o1.mgr = o2.empl_num AND  TARGET > 600.00;
+--
+SELECT ORDER_NUM, AMOUNT, DESCRIPTION  FROM ORDERS, PRODUCTS  WHERE MFR = MFR_ID AND PRODUCT = PRODUCT_ID;
+SELECT ORDER_NUM, AMOUNT, DESCRIPTION  FROM ORDERS o1 INNER JOIN PRODUCTS o2 ON o1.mfr = o2.mfr_id AND PRODUCT = PRODUCT_ID;
+--
+SELECT ORDER_NUM, AMOUNT, COMPANY, NAME FROM ORDERS, CUSTOMERS, SALESREPS WHERE CUST = CUST_NUM AND	  REP = EMPL_NUM AND  AMOUNT > 700;
+SELECT ORDER_NUM, AMOUNT, COMPANY, NAME FROM ORDERS t1 INNER JOIN CUSTOMERS t2 ON t1.cust = t2.cust_num INNER JOIN SALESREPS c1 ON
+c1.empl_num = t1.rep AND AMOUNT > 700;
+--
+SELECT ORDER_NUM, AMOUNT, COMPANY, NAME	FROM ORDERS, CUSTOMERS, SALESREPS WHERE CUST = CUST_NUM AND	  CUST_REP = EMPL_NUM AND	  AMOUNT > 700;
+SELECT ORDER_NUM, AMOUNT, COMPANY, NAME FROM ORDERS t1 INNER JOIN CUSTOMERS t2 ON t1.cust = t2.cust_num INNER JOIN SALESREPS c1 ON
+c1.empl_num = t2.cust_rep AND AMOUNT > 700;
+--
+SELECT ORDER_NUM, AMOUNT, COMPANY, NAME, CITY	FROM ORDERS, CUSTOMERS, SALESREPS, OFFICES WHERE CUST = CUST_NUM AND CUST_REP = EMPL_NUM AND 
+REP_OFFICE = OFFICE AND	  AMOUNT > 700;
+
+SELECT ORDER_NUM, AMOUNT, COMPANY, NAME, CITY	FROM ORDERS t1 INNER JOIN  CUSTOMERS t2 ON t1.cust=t2.cust_num INNER JOIN  SALESREPS t3 ON
+t3.empl_num = t2.cust_rep INNER JOIN  OFFICES t4 ON t4.office = t3.rep_office  AND	  AMOUNT > 700;
+--
+SELECT ORDER_NUM, AMOUNT, ORDER_DATE, NAME	FROM ORDERS, SALESREPS WHERE ORDER_DATE = HIRE_DATE;
+SELECT ORDER_NUM, AMOUNT, ORDER_DATE, NAME	FROM ORDERS t1 INNER JOIN  SALESREPS t2 ON t1.order_date=t2.hire_date;
+--
+SELECT NAME, QUOTA, CITY, TARGET	FROM SALESREPS, OFFICES WHERE QUOTA > TARGET;
+SELECT NAME, QUOTA, CITY, TARGET	FROM SALESREPS t1 INNER JOIN  OFFICES t2 ON   t1.quota > t2.target;
+SELECT 
+    name,
+    quota,
+    city,
+    target
+FROM
+    (
+        SELECT
+            *
+        FROM
+            salesreps
+            INNER JOIN offices ON salesreps.quota > offices.target
+    )
+WHERE
+    quota > target;
+     
```

Comment:
copy-past? :)
you have different style...

Reply:
Yes, this option is different, interesting for me